sql/pgwire_test.go

```diff
@@ -465,6 +465,12 @@ func TestPGPreparedExec(t *testing.T) {
 			},
 		},
 		{
+			"INSERT INTO d.T VALUES ($1) RETURNING 1",
```

Comment:
i feel like this test case should be in the query block rather than the exec block